services/director-v2/tests/unit/test_modules_dynamic_sidecar_docker_compose_specs.py

```diff
@@ -139,3 +140,34 @@ async def test_inject_resource_limits_and_reservations(
                 in spec["environment"]
             )
             assert f"{MEM_RESOURCE_LIMIT_KEY}={memory.limit}" in spec["environment"]
+
+
+@pytest.mark.parametrize(
+    "service_spec, expected_result",
+    [
+        pytest.param(
+            {"services": {"service-1": {}}},
+            {
+                "services": {
+                    "service-1": {"labels": [f"{LABEL_PREFIX_CONTAINER}.user.id=1"]}
+                }
+            },
+            id="single_service",
+        ),
+        pytest.param(
+            {"services": {"service-1": {}, "service-2": {}}},
+            {
+                "services": {
+                    "service-1": {"labels": [f"{LABEL_PREFIX_CONTAINER}.user.id=1"]},
+                    "service-2": {"labels": [f"{LABEL_PREFIX_CONTAINER}.user.id=1"]},
+                }
+            },
+            id="multiple_services",
+        ),
+    ],
+)
+async def test_update_container_labels(
+    service_spec: dict[str, Any], expected_result: dict[str, Any]
+):
+    docker_compose_specs._update_container_labels(service_spec, 1)
```

Comment:
ok I would like to discuss this stuff.
I see that this is going to add an exception to all the labels that we currently have:
e.g. the services have them (project, node and user as project_id, node_id, user_id), the tasks have them as well, the dask-sidecars are also setting them. Now this is going to be a different label using reverse notation (which I agree with @pcrespov ) but I disagree to have 2 different labelling at the same time. So either you follow the current labelling or we change them all. I'm ok if this is a 2 steps PR, but I find confusing and complicated to have 2 notations.

Reply:
As discussed using `SimcoreServiceDockerLabelKeys`, in the future this will automatically change when keys will be refactored.